assets/pickup-availability.js

```diff
@@ -57,7 +57,8 @@ if (!customElements.get('pickup-availability')) {
 
       document.body.appendChild(sectionInnerHTML.querySelector('pickup-availability-drawer'));
 
-      this.querySelector('button').addEventListener('click', (evt) => {
+      const button = this.querySelector('button');
+      if (button) button.addEventListener('click', (evt) => {
         document.querySelector('pickup-availability-drawer').show(evt.target);
       });
```

Comment:
Looks good 👍 I wonder since we are creating the const button twice, if we could create it once in the `fetch` and pass it as params 🤔 

But not sure if it would be readable/confusing

Reply:
Since one is in the `catch`, I'm not sure if it'd make sense 🤔 
I'm gonna leave it as is for now if that's ok 👍